### DISCUSSION TITLE
Looking for maintainers

### DISCUSSION THREAD

**Discussion by robacarp** [amberframework/amber#1286] [Announcements]
Amber remains one of the first landing places for new developers coming into crystal. It was a busy project for some time and has a wealth of technical implementation. At this time the developers (myself included) don't have the time to volunteer in the ways that the project needs.

If you're willing to participate in maintainership, please leave a message here or get in touch in Discord (`@rob` on [Crystal discord](https://discord.gg/YS7YvQy)). You don't need to be a grand master developer or have infinite free time -- the community around here is generally willing to contribute patches.

----

Edit: Any discussion about amber maintenance will now be on a dedicated [Amber discord server](https://discord.gg/wUJ4az5mVD).

**Comment by crimson-knight** [comment]
Hey, @robacarp I'm definitely interested!

**Comment by lexpank** [comment]
@robacarp I would be interested as well

**Comment by crimson-knight** [comment]
@AlexanderPankiv join us in the discord! That's the easiest way to take part

**Comment by HafeezRai** [comment]
@robacarp also interested
